types/socket.io-p2p/socket.io-p2p-tests.ts

```diff
@@ -0,0 +1,14 @@
+import * as P2P from "socket.io-p2p";
+import * as io from "socket.io-client";
```

Comment:
Since `P2P ` and `io` is class or function, it should be `import = require` here

Reply:
Do you have an article or blog post at hand that goes a bit into depth about when to use which form of import? Been wondering about the specifics for a while now.